iphone/Maps/LocalizedStrings/es-MX.lproj/Localizable.strings

```diff
@@ -1999,15 +1999,15 @@
 /* These translations are used for all type.highway.*.bridge. */
 "type.highway.footway.bridge" = "Puente";
 
-"type.highway.footway.demanding_alpine_hiking" = "Camino";
+"type.highway.footway.demanding_alpine_hiking" = "Path";
```

Comment:
@RedAuburn can you please check your environment and find the reason why translations here are overridden by default ones? You may try to debug the twine script.

Reply:
odd, i'll have a look